modules/apply-performance-profile-hosted-cluster.adoc

```diff
@@ -0,0 +1,187 @@
+====
+
+. Lists all `ConfigMaps` in the `clusters-democluster` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig="$MGMT_KUBECONFIG" get cm -n clusters-democluster
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                 DATA   AGE
+aggregator-client-ca                                 1      69m
+auth-config                                          1      68m
+aws-cloud-config                                     1      68m
+aws-ebs-csi-driver-trusted-ca-bundle                 1      66m
+...                                                  1      67m
+kubelet-client-ca                                    1      69m
+kubeletconfig-performance-democluster-us-east-1a     1      22m
+...
+ovnkube-identity-cm                                  2      66m
+performance-democluster-us-east-1a                   1      22m
+...                                                  
+tuned-performance-democluster-us-east-1a             1      22m
+----
++
+The output shows a kubeletconfig `kubeletconfig-performance-democluster-us-east-1a` and a performance profile `performance-democluster-us-east-1a` has been created. The Node Tuning Operator syncs the `Tuned` objects into the hosted cluster. You can verify which `Tuned` objects are defined and which profiles are applied to each node.
+
+. To list available secrets on the management cluster, run the following command:
++
+[source,terminal]
+----
+$ oc get secrets -n clusters
+----
++
+.Example output
+[source,terminal]
+----
+NAME                              TYPE                      DATA   AGE
+builder-dockercfg-25qpp           kubernetes.io/dockercfg   1      128m
+default-dockercfg-mkvlz           kubernetes.io/dockercfg   1      128m
+democluster-admin-kubeconfig      Opaque                    1      127m
+democluster-etcd-encryption-key   Opaque                    1      128m
+democluster-kubeadmin-password    Opaque                    1      126m
+democluster-pull-secret           Opaque                    1      128m
+deployer-dockercfg-8lfpd          kubernetes.io/dockercfg   1      128m
+----
+
+. Extract the `kubeconfig` file for the hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get secret <SECRET_NAME> -n clusters -o jsonpath='{.data.kubeconfig}' | base64 -d > hosted-cluster-kubeconfig
+----
++
+.Example
+[source,terminal]
+----
+$ oc get secret democluster-admin-kubeconfig -n clusters -o jsonpath='{.data.kubeconfig}' | base64 -d > hosted-cluster-kubeconfig
+----
+
+. Export the hosted cluster kubeconfig by running the following command:
++
+[source,terminal]
+----
+$ export HC_KUBECONFIG=<path_to_hosted-cluster-kubeconfig>
+----
+
+. Verify that the `kubeletconfig` is mirrored in the hosted cluster by running the following command:
```

Comment:
Not sure if kubeletconfig needs monospace or not - it doesn't have it earlier in this proc